tests/test_ami.py

```diff
@@ -193,3 +193,41 @@ def test_ami_cross_accounts(self):
         )
         resources = p.run()
         self.assertEqual(len(resources), 1)
+
+    def test_cel_unused_ami_false(self):
+        factory = self.replay_flight_data("test_unused_ami_false")
+        p = self.load_policy(
+            {
+                "name": "test-unused-ami",
+                "resource": "ami",
+                "filters": [
+                    {
+                        "type": "cel",
+                        "expr": """unused_image_filter(resource, "False")
```

Comment:
All of these single-line CEL expressions can use a single set of quotes. Triple quotation marks are used for multi-line strings in Python